### ISSUE TITLE
FMS Graph route finding algorithm

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#59]
## User story
1. As a SD
2. we need an algorithm to find the shortest route from point A to point B
3. So that the robot can find the shortest route to their destination

## Acceptance criteria
* Function should return a list of graph nodes an dedges that need to be traversed to get to the destination

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by nmarkert** [comment]
Implemented a graph search as an A* search in the `graph_search.py` file.
After searching a shortest path from node A to node B this path gets stored in a dictionary, so that if a path for the same combination is searched again, it is already available.

For the A* search we used the python-astar library: https://github.com/jrialland/python-astar

**Comment by UmangBR** [comment]
#35  describes the task to read in a file created from the SICK software and from this create a graph in our fms. This graph is stored in #37. The current issue uses the graph and finds the shortest path between two given points.
